client/src/components/CartItem/index.ts

```diff
@@ -0,0 +1 @@
+export { default as CartItem } from './CartItem';
```

Comment:
소소하게 아래처럼 작성해도 똑같이 import 해서 사용할 수 있답니다
```suggestion
export { default } from './CartItem';
```